spec/support/controller.rb

```diff
@@ -60,3 +60,11 @@ def stub_portal_controller
 def find_before_filters
   controller._process_action_callbacks.select{ |f| f.kind == :before }.map(&:filter)
 end
+
+# With strong_params, the params variable - and subhashes of params - in controllers
+# aren't really hashes. This method makes it easy to do things like
+# `expect(something).to have_received(:update).with(controller_params(name: "new name"))`
+# when we're dealing with stubbed objects in a test.
+def controller_params(hash)
+  ActionController::Parameters.new(hash).permit!
```

Comment:
Yeah it's a little more complicated now. Also, in controller specs, when you do something like `post :create, some_param: 1`, you get `params[:some_param] == "1"` now in the controller rather than `params[:some_param] == 1`. So parameters are being processed as they would in "real life" now in specs.